webapp/components/Comment.vue

```diff
@@ -39,18 +39,27 @@
         />
       </div>
       <div v-show="!openEditCommentMenu">
-        <div v-if="isCollapsed" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
+        <content-viewer
+          v-if="comment.content.length < 180"
+          :content="comment.content"
+          class="padding-left"
+        />
         <div
           v-show="comment.content !== comment.contentExcerpt && comment.content.length > 180"
           style="text-align: right;  margin-right: 20px; margin-top: -12px;"
         >
+          <content-viewer
+            v-if="isCollapsed"
+            :content="comment.contentExcerpt"
+            class="padding-left"
+          />
```

Comment:
@mattwr18 I don't understand why we have `content` twice now, and the `contentExcerpt` here..